### PR TITLE
Make small tweaks and proofread npe2 docs

### PR DESCRIPTION
**Pull request by DragaDoncila**
# Description
This PR makes small tweaks to the `npe2` docs and proofreads for grammar and spelling. Most of the changes are trivial but there are a few cases where I've swapped plugin names being used for commands and module names, to better fit the example at hand. @nclack can you please do a thorough review of these changes just to make sure I haven't messed anything up or misunderstood something.

## Type of change
- [x] This change is a documentation update

### CODE REVIEW

File: docs/plugins/npe2_migration_guide.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ recommend migrating to `npe2`. This guide will help you learn how!
 
 ## Migrating using the `npe2` command line tool
 
-`npe2` provides a command line interface to help convert a
+`npe2` provides a command line interface to help convert an original
 `napari-plugin-engine`-style plugin.
 
-### 1. Install the `npe2` command
+### 1. Install the `npe2` tool
```
Comment (nclack) [inline docs/plugins/npe2_migration_guide.md]:
```suggestion
### 1. Install the `npe2` command-line interface
```

**Review by nclack — APPROVED** [review]
Thanks so much!

File: docs/plugins/npe2_getting_started.md [hunk 2]
```diff
@@ -194,13 +194,14 @@ napari.manifest =
 
 The manifest file is specified relative to the submodule root path.
 So for the example it will be loaded from:
-`<path/to/npe2-tester>/napari.yaml`.
+`<path/to/npe2-tester>/npe2_tester/napari.yaml`.
```
Comment (sofroniewn) [inline docs/plugins/npe2_getting_started.md:197]: should this be `.yml` too?

Comment (nclack) [reply docs/plugins/npe2_getting_started.md:197]: You're right, it's `.yml` right now. I'd rather change it to `.yaml` on the cookiecutter side though (napari/cookiecutter-napari-plugin#76).

Comment (sofroniewn) [reply docs/plugins/npe2_getting_started.md:197]: ok that's fine. i guess this can be merge now then

**Comment by sofroniewn** [comment]
i'll merge this tomorrow morning unless i hear otherwise. thanks for fixes!

**Comment by tlambert03** [comment]
fwiw, #3916 will supercede this

**Comment by sofroniewn** [comment]
> fwiw, #3916 will supercede this

Ah ok then probably not worth merging this now and creating unnecessary merge conflicts. Maybe we focus on reviewing that and making sure any changes here are reflect. If others disagree though please let me know and I can merge this and then we'll fix any merge conflicts as need. thanks!!!

**Comment by tlambert03** [comment]
> making sure any changes here are reflect.

fwiw, I did go through the diff here and make sure that all changes are reflected in that PR (inasumuch as the section still exists)

**Comment by DragaDoncila** [comment]
As @tlambert03 mentioned, this is now superseded by #3916 which I've just reviewed, so I will close this PR in favour of that one.

**Pull request closed without merging**